docs/shipping/shipping-method-api.md

```diff
@@ -76,7 +76,7 @@ if ( ! class_exists( 'WC_Your_Shipping_Method' ) ) {
 
 ## Defining settings/options
 
-You can then define your options using the settings API. In the snippets above you'll notice we init_form_fields and init_settings. These load up the settings API. To see how to add settings, see [WooCommerce settings API](https://woo.com/document/settings-api/).
+You can then define your options using the settings API. In the snippets above you'll notice we init_form_fields and init_settings. These load up the settings API. To see how to add settings, see [WooCommerce settings API](https://github.com/woocommerce/woocommerce/blob/trunk/docs/extension-development/settings-api.md).
```

Comment:
@babarkhuroo Thanks for submitting the PR. Could you make this a relative link so that when it's ingested into the docs site it links to the content on the docs site?

```suggestion
You can then define your options using the settings API. In the snippets above you'll notice we init_form_fields and init_settings. These load up the settings API. To see how to add settings, see [WooCommerce settings API](../extension-development/settings-api.md).
```